student-library/src/main/java/tds/student/sql/data/AccList.java

```diff
@@ -8,36 +8,27 @@
  ******************************************************************************/
 package tds.student.sql.data;
 
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-
+import TDS.Shared.Exceptions.ReadOnlyException;
 import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.IteratorUtils;
 import org.apache.commons.collections.Predicate;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import tds.itemrenderer.data.AccLookup;
-import AIR.Common.DB.results.DbResultRecord;
-import TDS.Shared.Data.ColumnResultSet;
-import TDS.Shared.Exceptions.ReadOnlyException;
-import TDS.Shared.Exceptions.ReturnStatusException;
 
-public class AccList extends ArrayList<Data>
```

Comment:
This wasn't going to fly with simple serialization/deserialization.  I refactored the ArrayList<Data> into a new "data" property.  This will require changes in Student.